src/api/app/models/user.rb

```diff
@@ -349,6 +349,7 @@ def password=(value)
   def update_password(pass)
```

Comment:
the changes in this method as this is exactly the same case:

> it doesn't make sense to require an API user to provide an additional XML element called

This is only used by api users and that's the reason why you duplicate the password here, so exactly the same thing.